plasmapy/diagnostics/langmuir.py

```diff
@@ -209,50 +209,51 @@ def swept_probe_analysis(probe_characteristic, probe_area, gas,
     gas : ~astropy.units.Quantity
         The (mean) mass of the background gas in atomic mass units.
 
-    visualize : bool, Optional
+    visualize : bool, optional
         Can be used to plot the characteristic and the obtained parameters.
         Default is False.
 
-    plot_electron_fit : bool, Optional
+    plot_electron_fit : bool, optional
         If True, the fit of the electron current in the exponential section is
         shown. Default is False.
 
-    plot_EEDF : bool, Optional
+    plot_EEDF : bool, optional
         If True, the EEDF is computed and shown. Default is False.
 
     Returns
     -------
-    A dictionary with the following entries:
 
-    'T_e' : ~astropy.units.Quantity
+    Results are returned as Dictionary
```

Comment:
This is absolutely not something we should change or do right now, just a loose idea I just had.

Instead of a `dict`, we could return a `PlasmaBlob`.

Reply:
Absolutely, but I'd prefer the user to have the option which format is returned.

Reply:
Fair enough. I was thinking this isn't such a big deal as in essence you either get dictionary variable access
```python
result['T_e']
```
or namedtuple variable access
```python
result.T_e
```
The latter seems to me to be cleaner.